app/controllers/admins_controller.rb

```diff
@@ -1,34 +1,66 @@
 class AdminsController < ApplicationController
-  before_action :authenticate_user!
-    
+  # restrict methods (except signup) in this file to admin only
+  before_action :authenticate_user!, except: [:signup]
+  
+  # load add items procedures and facilities to pass into index view ----- will chage to pending bills once that is working 
   def index
-    @items = Item.where(status: 'pending')
-    @procedures = Procedure.where(status: 'pending')
+    @items = Item.all
+    @procedures = Procedure.all
+    @facilities = Facility.all
   end
 
+  # not going to need once approve bill is working
```

Comment:
They could be deleted I was just leaving them for now in case we needed them again.  Either way is good with me.

Reply:
If you think you might need them again, then I think that commenting them out makes the most sense for now. I just didn't want to leave dangling code with the good intention of cleaning it up later (because we all know how that goes!).

Reply:
deleted unneeded methods.